src/index.js

```diff
@@ -48,44 +36,28 @@ class ContentLoader extends React.Component<Props, State> {
     className: '',
   }
 
-  constructor(props: Props) {
-    super(props)
-
-    this.state = {
-      style: props.style,
-      type: props.type,
-      speed: props.speed,
-      width: props.width,
-      height: props.height,
-      primaryColor: props.primaryColor,
-      secondaryColor: props.secondaryColor,
-      preserveAspectRatio: props.preserveAspectRatio,
-      className: props.className,
-    }
-  }
-
   render() {
     if (this.props.children) {
-      return <Wrap {...this.state}>{this.props.children}</Wrap>
+      return <Wrap {...this.props}>{this.props.children}</Wrap>
     }
 
     if (!this.props.children) {
```

Comment:
There is one thing that I would like to change and I've always forgotten... 
This last `if () {}` isn't necessary, could you please remove it?